<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="54f88fed-c4fb-4115-8bd9-97546f2e1a20" name="Jamie@JAMIE-VAIO 2012-09-01 13:44:27" runUser="JAMIE-VAIO\Jamie" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="67bacfd6-e3f0-4c66-ad16-3c0a8f16e5e2">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Jamie_JAMIE-VAIO 2012-09-01 13_44_27" />
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-09-01T13:44:27.6195372+01:00" queuing="2012-09-01T13:44:27.8985532+01:00" start="2012-09-01T13:44:28.0645627+01:00" finish="2012-09-01T13:44:28.2915757+01:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="Throws_Exception_If_Called_Before_ExecuteService" storage="c:\repositories\servicemanager\src\servicemanager.tests\bin\debug\servicemanager.tests.dll" id="8e934a81-44cf-0c23-c447-ffb0b77d4832">
      <Execution timeOut="1800000" id="45160b6f-f61e-40c8-80ff-81e24d72026c" />
      <TestMethod codeBase="C:/Repositories/ServiceManager/src/ServiceManager.Tests/bin/Debug/ServiceManager.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="ServiceManager.Tests.Given_ServiceManager_Instance+When_IfServiceFailsThen_Invoked, ServiceManager.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Throws_Exception_If_Called_Before_ExecuteService" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="8e934a81-44cf-0c23-c447-ffb0b77d4832" executionId="45160b6f-f61e-40c8-80ff-81e24d72026c" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="45160b6f-f61e-40c8-80ff-81e24d72026c" testId="8e934a81-44cf-0c23-c447-ffb0b77d4832" testName="Throws_Exception_If_Called_Before_ExecuteService" computerName="JAMIE-VAIO" duration="00:00:00.0735467" startTime="2012-09-01T13:44:28.1085652+01:00" endTime="2012-09-01T13:44:28.2365725+01:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="45160b6f-f61e-40c8-80ff-81e24d72026c">
      <Output>
        <ErrorInfo>
          <Message>Test method ServiceManager.Tests.Given_ServiceManager_Instance+When_IfServiceFailsThen_Invoked.Throws_Exception_If_Called_Before_ExecuteService threw exception: 
System.Exception: You must call the ExecService method before IfServiceFailsThen</Message>
          <StackTrace>    at Service_Manager.ServiceManager.IfServiceFailsThen[TResult](Func`1 action) in C:\Repositories\ServiceManager\src\ServiceManager\ServiceManager.cs:line 142
   at ServiceManager.Tests.Given_ServiceManager_Instance.When_IfServiceFailsThen_Invoked.Throws_Exception_If_Called_Before_ExecuteService() in C:\Repositories\ServiceManager\src\ServiceManager.Tests\ServiceManager.Tests.cs:line 65
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>